presc/copies/copying.py

```diff
@@ -3,6 +3,29 @@
 
 
 class ClassifierCopy:
+    """Represents a classifier copy and its associated sampling method of choice.
+
+    Each instance wraps the original ML classifier with a ML classifier copy and
+    the sampling method to carry out the copy. Class methods allow to carry out
```

Comment:
Super nit: "class method" means something different in Python. Would be best to drop the word "class" here.